include/utilities.h

```diff
@@ -459,4 +459,15 @@ void smart_delete (SmartPointer<TYPE> &sp)
     }
 }
 
+
+template <typename VEC>
+void vector_shift(VEC &in_vec, double a_scalar)
+{
+  for (auto i : in_vec.locally_owned_elements())
```

Comment:
@asartori86 , what do you think? Should we ask our deal2lkit users to have a C++11 compiler?